### ISSUE TITLE
opam.ocaml.org: stacked navs are fat and ugly

### ISSUE THREAD

**Issue by dsheets** [ocaml/v2.ocaml.org#522]
![stacked_navs_fat_ugly](https://cloud.githubusercontent.com/assets/59918/4024323/f6bd3fba-2bc2-11e4-90e5-1dddc4a8ba8d.png)

**Comment by Chris00** [comment]
One way to deal with this is to transform the lower nav into a menu.  The 3 columns on the [leading page](http://ocaml.github.io/platform-dev/) are half filled anyway and [many](http://ocaml.github.io/platform-dev/packages/0install/) [pages](http://ocaml.github.io/platform-dev/about.html) have too long lines — harder to read.  For the [documentation](http://ocaml.github.io/platform-dev/doc/Quick_Install.html), I believe the menu (coming from the current nav) could "open" to have the present menu (which contains too large vertical spaces) as a submenu.  What do you think?

**Comment by samoht** [comment]
I think any kind of styling simplification is more than welcome :-)

We can also have a way to hide the main nav bar (I'm pretty sure you don't want any kind of nav when you are browsing documentation for instance).

**Comment by Chris00** [comment]
> I'm pretty sure you don't want any kind of nav when you are browsing documentation for instance

Yes.  A running header would still be nice in order to ease navigation.
